app-ios/Sources/ContributorFeature/ContributorView.swift

```diff
@@ -1,41 +1,151 @@
+public let contributorReducer = Reducer<ContributorState, ContributorAction, ContributorEnvironment> { state, action, environment in
+    switch action {
+    case .refresh:
+        return .run { @MainActor subscriber in
+            for try await result: [Contributor] in environment.contributorsRepository.contributors().stream() {
+                await subscriber.send(
+                    .refreshResponse(
+                        TaskResult {
+                            result
+                        }
+                    )
+                )
+            }
+        }
+        .receive(on: DispatchQueue.main.eraseToAnyScheduler())
+        .eraseToEffect()
+    case .refreshResponse(.success(let contributors)):
+        state.contributors = contributors
+        return .none
+    case .refreshResponse(.failure):
+        return .none
+    }
 }
 
 public struct ContributorView: View {
+
+    @Environment(\.openURL) var openURL
+
     private let store: Store<ContributorState, ContributorAction>
 
     public init(store: Store<ContributorState, ContributorAction>) {
         self.store = store
+
+        UITabBar.appearance().barTintColor = AssetColors.surface.color
     }
 
     public var body: some View {
-        Text("TODO: Contributor")
+        WithViewStore(store) { viewStore in
+            List(viewStore.contributors, id: \.id) { contributor in
+                ContributorItemView(contributor: contributor, action: {
+                    guard let profileUrl = contributor.profileUrl else { return }
+                    if let url = URL(string: profileUrl) {
+                        openURL(url)
+                    }
+                })
+                .listRowInsets(EdgeInsets())
+                .listRowBackground(AssetColors.background.swiftUIColor)
+                .listRowSeparator(.hidden)
+            }
+            .task {
+                await viewStore.send(.refresh).finish()
+            }
+            .listStyle(PlainListStyle())
+            .navigationTitle(L10n.Contributors.title)
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+}
+
+struct ContributorItemView: View {
+
+    let contributor: Contributor
+    var action: () -> Void
+
+    var body: some View {
+        ZStack {
+            HStack(spacing: 16) {
+                AsyncImage(url: URL(string: contributor.iconUrl)) { image in
+                    image.resizable()
+                } placeholder: {
+                    Color(.systemGray4)
+                }
+                .frame(width: 60, height: 60)
+                .clipShape(Circle())
+                Text(contributor.username)
+                    .font(Font.system(size: 16, weight: .bold, design: .default))
+                    .foregroundColor(AssetColors.onBackground.swiftUIColor)
+                Spacer()
+            }
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)
+
+            Button {
+                action()
+            } label: {
```

Comment:
To open url without NavigationLink (push navigation), use Button.

Reply:
How about use `NavigationLink` with `isActive` ? 👀 

My idea is wrap this view with Button in `ContributorView` and when tap this view, toggle `isActive` state.

If it seems difficult, it is okey! I'll create another issue about this 👍🏼 

Example: 
https://github.com/DroidKaigi/conference-app-2021/blob/e9540bf4e8bbf9530a08db8c877cf3e5bb4eb7c4/ios/Sources/TimetableFeature/List/TimetableLoadedView.swift#L68-L83

Reply:
I tried. But I couldn't prevent push navigation with NavigationLink.

To modify row color and hide arrow of trailing side, It is better to use Button than NavigationLink for now, I thought.

Reply:
Okey!
I will try to challenge it with another issue or PR. Thank you for trying!